### DISCUSSION TITLE
Help- locked out, no 2FA, but mobile app works

### DISCUSSION THREAD

**Discussion by Bermy1988** [community/community#132590] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a rather unusual situation here- I had to create a separate account to make this post.  I have an account on here, made with an email account that I don't check very frequently.  I haven't been using it very much until now, and I didn't see the emails telling me to set up 2FA.  The way I'd sign in on my laptop/browser was by confirming the notification on my mobile app on my tablet.  The problem is that the tablet wasn't always turned on, and a few days ago it stopped sending the notifications to the tablet entirely.  I never downloaded the recovery codes either.  I am able to access my account via the mobile app, but it only has certain features.   When I try to contact support without logging in, it's a chatbot with pre-written responses, all of which presume that I have 2FA set up.  From what I can tell, the problem is the notifications not arriving when they get sent to my tablet/mobile app.  Does anyone know something I can try?

**Comment by Davdadev** [comment]
yes, I have experienced this too when logging in. Is this the screen you are stuck on?
![Screenshot_18-7-2024_112353_github com](https://github.com/user-attachments/assets/40a2511e-e8e7-4fd4-b9f2-b6d1bac0e860)

**Reply by Bermy1988** [reply]
Yes, that's it.

**Reply by Davdadev** [reply]
do u have a recovery code

**Reply by aaronh197** [reply]
Google has data.
